pkgs/os-specific/linux/keyutils/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     ./conf-symlink.patch
     # This patch solves a duplicate symbol error when building with a clang stdenv
     # Before removing this patch, please ensure the package still builds by running eg.
-    # nix-build -E 'with import ./. {}; pkgs.keyutils.override { stdenv = pkgs.llvmPackages_latest.stdenv; }'
+    # nix-build -E 'with import ./. {}; pkgs.keyutils.override { stdenv = pkgs.llvmPackages_14.stdenv; }'
```

Comment:
Default seems to make more sense here?

Reply:
Agreed.  I've changed it to `pkgs.clangStdenv`.